apps/cookbook/src/app/showcase/dropdown-showcase/dropdown-showcase.component.html

```diff
@@ -2,33 +2,70 @@ <h2>Simple</h2>
 <h3>Default</h3>
 <cookbook-example-viewer [html]="defaultExample.template">
   <div>
-    <cookbook-dropdown-example-default #defaultExample></cookbook-dropdown-example-default>
+    <cookbook-dropdown-example-default
+      [size]="size"
+      #defaultExample
+    ></cookbook-dropdown-example-default>
   </div>
 </cookbook-example-viewer>
+<fieldset>
```

Comment:
Consider exctracting to component and use in to `apps\cookbook\src\app\examples\dropdown-example\dropdown-example.component.html` to make it possible to configure size when in pop-out mode